pages/upgrade/graylog-2.3.rst

```diff
@@ -0,0 +1,87 @@
+**************************
+Upgrading to Graylog 2.3.x
+**************************
+
+.. _upgrade-from-22-to-23:
+
+This file only contains the upgrade note for the upcoming release.
+Please see `our documentation <http://docs.graylog.org/en/latest/pages/upgrade.html>`_
+for the complete upgrade notes.
+
+Graylog switches to Elasticsearch HTTP client
+=============================================
+
+In all prior versions, Graylog used the Elasticsearch node client to connect to an Elasticsearch cluster, which was acting as a client-only Elasticsearch node. For compatibility reasons of the used binary transfer protocol, the range of Elasticsearch versions Graylog could connect to was limited. For more information and differences between the different ways to connect to Elasticsearch, you can check the `Elasticsearch documentation <https://www.elastic.co/guide/en/elasticsearch/guide/current/_talking_to_elasticsearch.html>`_.
```

Comment:
The URL should point to a specific version: https://www.elastic.co/guide/en/elasticsearch/guide/5.4/_talking_to_elasticsearch.html